Barkane/Assets/Scripts/UI/GameManager.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class GameManager : MonoBehaviour
```

Comment:
Should probably be called MainMenuManager, GameManager implies it's gameplay related